source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
+                        <p>For describing the <gi scheme="MEI">scoreFormat</gi> the value list of MARC21 (see “20 - Format of music (006/03)”, https://www.loc.gov/marc/bibliographic/bd008m.html) should be used. If this value list is not sufficient, however, it is recommended to design an own classification (see <gi scheme="MEI">classification</gi>).</p>
+                           <p><gi scheme="MEI">scoreFormat</gi> can similarly be used for classification in <gi scheme="MEI">expression</gi>.</p>
+                              <!--<p>Internal note: Even <gi scheme="MEI">physDesc</gi> is part of the MEI.header module, the parent elements of <gi scheme="MEI">physDesc</gi> (<gi scheme="MEI">manifestation</gi>, <gi scheme="MEI">item</gi>) are part of MEI.frbr. So the use of the module MEI.frbr is mandatory for the use of <gi scheme="MEI">scoreFormat</gi>.</p>-->
                   </div>
-                  <div xml:id="patches" type="div4">
-                     <head>Patches</head>
-                     <p>Sometimes, manuscripts (but also prints) are subject to modifications that do not change the textual content, but the actual physical item. Typical examples for this are patches glued on a page, or cutouts. Both these situations can be encoded inside <gi scheme="MEI">foliaDesc</gi>.</p>
-                     <p>A patch is an additional writing surface attached to one of the sides of a <gi scheme="MEI">folium</gi> or <gi scheme="MEI">bifolium</gi>:</p>
+                  <div xml:id="bindingDesc" type="div4">
+                     <head>Describing the Binding</head>
+                  <p></p>
                      <p>
                         <specList>
-                           <specDesc key="patch"/>
+                           <specDesc key="bindingDesc"/>
+                           <specDesc key="binding"/>
                         </specList>
                      </p>
-                     <p>The <gi scheme="MEI">patch</gi> element is placed inside the <gi scheme="MEI">folium</gi> or <gi scheme="MEI">bifolium</gi> to which it is attached. To which side of this parent it is attached is specified using the (required) <att>attached.to</att> attribute:</p>
+                     <p></p>
+                  </div>
+                  <div xml:id="foliadesc" type="div4">
+                     <head>Description of folia</head>
+                     <p>While many other elements within <gi scheme="MEI">physDesc</gi> describe specific features of manuscripts and prints in prose, <gi scheme="MEI">foliaDesc</gi> is intended to be processable. It provides information about the binding of a manuscript or print and the layout of its pages. The most important elements used are:</p>
```

Comment:
```suggestion
                     <p>While many other elements within <gi scheme="MEI">physDesc</gi> describe specific features of manuscripts and prints in prose, <gi scheme="MEI">foliaDesc</gi> is intended provide a structured description. It provides information about the binding of a manuscript or print and the layout of its pages. The most important elements used are:</p>
```

Reply:
It may be out of scope, but you may wish to describe the use of `bifolium` for describing gatherings. See: https://spcoll.library.uvic.ca/Digit/lydgate/lydgatems/pages/manuscript/collation/collation.htm